joomla/joomla-cms
# [com_cache] cache view: Add missing tooltip in search

#### Summary of Changes

Simple PR to add missing tootip to com_cache search filter.
#### Testing Instructions

Code review.

or
1. Apply patch
2. Go to "System" -> "Clear Cache", check the tootip in the search field.
